packages/openapi-generator/src/parser/operation.ts

```diff
@@ -47,13 +71,29 @@ export function parseParameters(
 ): OpenApiParameter[] {
   // TODO: What if this is a reference? What does OpenApi do?
   // TODO: What about oneof and other operations?
-  return (
-    operation.parameters
-      ?.map(param => resolveObject(param, refs))
-      .map(param => ({
-        ...param,
-        type: getType(resolveObject(param.schema, refs)?.type?.toString())
-      })) || []
+  return filterDuplicates(
+    operation.parameters?.map(param => resolveObject(param, refs)) || []
+  ).map(param => ({
+    ...param,
+    type: getType(resolveObject(param.schema, refs)?.type?.toString())
+  }));
+}
+
+export function filterDuplicates(
```

Comment:
I would prefer to have a removeDuplicates() method in the array util where you can hand in a custom comparator function `comperator(a:T,b:T):boolean`. Then we can reuse it better.

Reply:
Good idea. Done.